spec/models/group_spec.rb

```diff
@@ -57,6 +57,11 @@
       group.status = 'locked'
       expect(group.valid?).to be_falsey
     end
+    it 'can only be locked if all memberships are locked' do
+      group = FactoryGirl.create(:finalizing_group)
```

Comment:
The connection between this setup and the test conditions is unclear, probably because it's all included in the factory. Can we be more explicit?